src/tabmat/ext/cat_split_helpers-tmpl.cpp

```diff
@@ -15,16 +15,16 @@ void _transpose_matvec_${type}(
 ) {
     int num_threads = omp_get_max_threads();
     std::vector<F> all_res(num_threads * res_size, 0.0);
-    #pragma omp parallel shared(all_res) 
+    #pragma omp parallel shared(all_res)
```

Comment:
I incorporated the changes from https://github.com/Quantco/tabmat/pull/348 here. @adityagoel4512 @stanmart would you mind having a quick look at the changes?

Reply:
Looks fine from my perspective. The nightly failures look unrelated to the change.

Reply:
Thanks for checking. Yes, I think the nightly failures are related to the install failure for 3.9 in https://github.com/Quantco/tabmat/issues/350 - a separate issue.